docs/source/app-dev/daml-lf-translation.rst

```diff
@@ -245,3 +265,23 @@ These translate to the DAML-LF record declarations:
 
 	record DoNothing ↦ {}
 	record Transfer ↦ { newOwner: Party }
+
+Generic template instantiati
+
+Generic template instantiations
+===============================
+
+In the context of the templates ``Iou`` and ``Proposal`` above, the generic template instantiation
+
+.. literalinclude:: code-snippets/daml-lf-translation.daml
+   :language: daml
+   :start-after: -- start code snippet: generic template instantiation
+   :end-before: -- end code snippet: generic template instantiation
+
+which does not produce a DAML data definition but only the DAML-LF record declaration:
```

Comment:
The claim that we don't generate a DAML data type is a deliberate lie. I'll change this to produce a type synonym in a follow up PR and will document that then.